criu/config.c

```diff
@@ -1082,7 +1082,7 @@ int check_options(void)
 	}
 
 	if (opts.restore_on_demand)
-		pr_info("Restore using sys_mmap");
+		pr_info("Restore using sys_mmap\n");
```

Comment:
this should be in another patch

Reply:
Sorry, I don’t know what "another patch" mean. Should I merge it into another branch?